actionpack/lib/abstract_controller/rendering.rb

```diff
@@ -64,12 +64,12 @@ def rendered_format
     # You can overwrite this configuration per controller.
     def view_assigns
       protected_vars = _protected_ivars
-      variables      = instance_variables
```

Comment:
Curious what if would have looked like as `variables = instance_variables - _protected_ivars`?

Reply:
That is indeed faster and clearer. I will open a PR to change that.